src/server/src/main/java/io/cassandrareaper/resources/ClusterResource.java

```diff
@@ -281,11 +294,16 @@ private Response addOrUpdateCluster(
   private Cluster updateClusterSeeds(Cluster cluster, String seedHosts) throws ReaperException {
     Set<String> newSeeds = parseSeedHosts(seedHosts);
     try {
-      Set<String> previousNodes = ImmutableSet.copyOf(clusterFacade.getLiveNodes(cluster));
+      Set<String> previousNodes = Collections.emptySet();
+      try {
+        previousNodes = ImmutableSet.copyOf(clusterFacade.getLiveNodes(cluster));
+      } catch (ReaperException expected) {
```

Comment:
What ReaperExceptions would be thrown and caught here. Can't say I'm comfortable with this approach, it should at a minimum check the cluster state.